deviceModeInit.js

```diff
@@ -153,32 +159,43 @@ var rudderTracking = (function () {
 
   function checkCartAttributes(cart) {
     const { attributes } = cart;
-    if (attributes?.rudderAnonymousId && attributes?.rudderUpdatedAt) {
-      return true;
+    if (attributes?.rudderAnonymousId) {
+      return false;
     }
-    return false;
+    return true;
   }
 
+  function updateTimeStampFoIdentifierEvent() {
+    const cookieOptions = {
+      action: "set",
+      expire_hr: 2,
+      name: "rs_shopify_cart_identified_at",
+      value: `${Date.now()}`
+    }
+    cookie_action(cookieOptions);
+  }
   function updateCartAttribute() {
     const anonymousId = rudderanalytics.getAnonymousId();
     return rs$.post(
       window.Shopify.routes.root + "cart/update.json",
       {
         attributes: {
           rudderAnonymousId: anonymousId,
-          rudderUpdatedAt: Date.now(),
         },
       },
       undefined,
       "json"
     );
   }
-
-  function checkUpdateTime(cart) {
+  function checkUpdateTime() {
```

Comment:
Function name is not synonymous to behaviour please address

Reply:
Done